docs/README.md

```diff
@@ -20,6 +20,7 @@
   - [Initialize the Project](###initialize-the-project)
   - [Register the DID](###register-the-did)
   - [Build and Start the Project](###build-and-start-the-project)
+  - [Updates to Credential Schema](###updates-to-credential-schema)
```

Comment:
The table of content links (all of them) are broken.  I believe the proper syntax for a TOC link is to use a single `#`.  Please fix.